compiler/src/org.graalvm.compiler.truffle.compiler/src/org/graalvm/compiler/truffle/compiler/TruffleCompilerImpl.java

```diff
@@ -420,6 +420,11 @@ public int getInfopointsCount() {
             return res;
         }
 
+        @Override
+        public Infopoint[] getFullInfopoints() {
```

Comment:
Right - I need to document that you won't get infopoint comments for `libgraal`. That's why I made this interface method default. I saw that `libgraal` passes infopoints through as strings, possibly to avoid having to implement a lot of objects in the interface.